string mission_name
string points_of_interest
---
